README.md

```diff
@@ -24,3 +23,21 @@ At the moment, the only way to go back to gallery screen is to hit the
    - `url`: the URL of the notebook to render.
    - `repo_url`: the URL of the repository serving as source.
    - `image_url`: the URL of the picture to use as thumbnail.
+
+## Deploying you own gallery
+
+The voila gallery is built as a plugin for [The Littlest JupyterHub (TLJH)](https://tljh.jupyter.org). To deploy your own instance:
+
+1. Fork the gallery repo: https://github.com/voila-gallery/gallery
+2. Edit the `tljh-voila-gallery/tljh_voila_gallery/gallery.yaml` file with your own set of examples
+3. Follow [one of the tutorials to install TLJH](https://tljh.jupyter.org/en/latest/#installation)
+4. At the step asking for user data, use the following command:
+
+```
+#!/bin/bash
+curl https://raw.githubusercontent.com/jupyterhub/the-littlest-jupyterhub/master/bootstrap/bootstrap.py \
+ | sudo python3 - \
+   --plugin git+https://github.com/<your-username>/gallery@master#"egg=tljh-voila-gallery&subdirectory=tljh-voila-gallery"
```

Comment:
Is there a way to make this shorter? I think the trick here is that it's a subdirectory of the repo.

Reply:
We can publish to PyPI... :D

Reply:
ah, that could be one solution for the "official" gallery.
Might become a bit tedious when deploying your own from a GitHub fork though.